Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -76,6 +82,8 @@
 
 - type: reaction
   id: B52
+  requiredMixerCategories:
+  - Shake
```

Comment:
The B52 is a layered drink, so don't think shaking is appropriate.

![7428618800b0433caae86bb33e3c98f5](https://github.com/space-wizards/space-station-14/assets/85356/228d3590-86d4-4b5d-9de5-d08b4c91c797)